torch_np/tests/numpy_tests/core/test_numeric.py

```diff
@@ -1251,78 +1251,60 @@ def test_failed_itemsetting(self):
             np.fromiter(iterable, dtype=np.dtype((int, 2)))
 
 
-@pytest.mark.xfail(reason="TODO")
-class TestNonzero:
+class TestNonzeroAndCountNonzero:
+    def test_count_nonzero_list(self):
+        lst = [[0, 1, 2, 3], [1, 0, 0, 6]]
+        assert np.count_nonzero(lst) == 5
+        assert_array_equal(np.count_nonzero(lst, axis=0), np.array([1, 1, 1, 2]))
+        assert_array_equal(np.count_nonzero(lst, axis=1), np.array([3, 2]))
+
     def test_nonzero_trivial(self):
         assert_equal(np.count_nonzero(np.array([])), 0)
-        assert_equal(np.count_nonzero(np.array([], dtype='?')), 0)
+        assert_equal(np.count_nonzero(np.array([], dtype="?")), 0)
```

Comment:
If this comes from black, it'd be good to separate blackening of files into their own PRs to avoid longer diffs.

Reply:
Yes indeed. I'm going to refrain from commenting about black et al, at all.